src/components/Explore/TokenDetails/TokenDetail.tsx

```diff
@@ -70,7 +71,7 @@ const ContractAddress = styled.button`
   cursor: pointer;
 
   &:hover {
-    color: ${({ theme }) => theme.textSecondary};
+    color: ${({ theme }) => darken(0.1, theme.textPrimary)};
```

Comment:
check if this is what they want

Reply:
I'm just gonna push this for now so that Cal can do some UI polish stuff -- they don't have a design for this yet from the Figma, so Cal can add it to the UI feedback when ready